microsoft-edge/webview2/how-to/remote-debugging-xbox.md

```diff
@@ -0,0 +1,121 @@
+
+1.  In Microsoft Edge, go to `edge://inspect`.  The **Inspect with Edge Developer Tools** page opens:
+
+    ![DevTools Inspect utility page](./remote-debugging-xbox-images/inspect-devtools-page-supported.png)
+
+1.  Enable Developer Mode on Xbox.  See [Xbox One Developer Mode activation](/windows/uwp/xbox-apps/devkit-activation).
+
+1.  In Microsoft Edge, go to `https://<System IP>:11443`, where `<System IP>` is replaced by your Xbox System OS IP.  The **Confirm security settings** page is displayed, in the **Windows Device Portal** tab:
+
+    ![The 'Confirm security settings' page in the 'Windows Device Portal' tab](./remote-debugging-xbox-images/open-device-portal.png)
+
+    An SSL cert error appears:
+
+    ![SSL Error](./remote-debugging-xbox-images/xbox_ssl_error.png)
+
+1.  Click the **Advanced** button. <!--todo: state the system result-->
+
+1.  Click **Continue**<!--todo: button?--> to `<System IP>`,<!--todo: clarify "Click Continue to System IP"--> where `<System IP>` is replaced by your Xbox System OS IP.
+
+    Xbox Device Portal opens:
+
+    ![Xbox Device Portal](./remote-debugging-xbox-images/unsecure-xbox-device-portal.png)
+
+1.  Go to `https://<System IP>:11443/config/rootcertificate`, where `<System IP>` is replaced by your Xbox System OS IP.  This downloads a `rootcertificate.cer` file:
+
+    ![Root Certificate file listed in Downloads dialog of Microsoft Edge](./remote-debugging-xbox-images/root-certificate.png)
+
+1.  Open the downloaded `rootcertificate.cer` file.  A security warning dialog box opens:
+
+    ![Security warning dialog box](./remote-debugging-xbox-images/open-file-security-warning.png)
+
+1.  Click the **Open** button.  A **Certificate** dialog opens:
+
+    ![The Certificate dialog](./remote-debugging-xbox-images/certificate.png)
+
+1.  Click the **Install Certificate** button.  A **Welcome to the Certificate Import Wizard** dialog opens:
+
+    ![The 'Welcome to the Certificate Import Wizard' dialog](./remote-debugging-xbox-images/current-user-certificate.png)
+
+1.  Select the **Current User** option button, and then click the **Next** button.
+
+    The Certificate Import Wizard opens:
+
+    ![Certificate Import Wizard](./remote-debugging-xbox-images/certificate-import-wizard.png)
+
+1.  Select the option **Place all certificates in the following store**, click the **Browse** button, and then select **Trusted Root Certification Authorities**.
+
+1.  Click the **Next** button.
+
+    A **Security Warning** dialog opens:
+
+    ![Security Warning dialog](./remote-debugging-xbox-images/security-warning.png)
+
+1.  Click the **Yes** button.
+
+1.  Restart Microsoft Edge.
+
+1.  Go to `https://<System IP>:11443`, where `<System IP>` is replaced by your Xbox System OS IP, and verify that the connection is indicated as secure:<!--todo: clarify exactly how to verify that the connection is indicated as secure; and add red box on png.  do we mean https in Address bar? specify -->
```

Comment:
@maurawinstanley - the png shows as "Not secure":
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/remote-debugging-xbox-images/unsecure-xbox-device-portal.png

Clarify specifically how to verify that the connection is indicated as secure; and add red box on png. do we mean https in Address bar? specify.  What exactly/concretely are we telling user to do?

Reply:
Rendered section: 
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/remote-debugging-xbox?branch=pr-en-us-2846#attach-devtools-remotely-to-an-xbox-webview2-winui-2-uwp-app
to see if png was changed, etc.